doc/classes/Geometry2D.xml

```diff
@@ -125,8 +125,28 @@
 			<param index="2" name="from_b" type="Vector2" />
 			<param index="3" name="dir_b" type="Vector2" />
 			<description>
-				Checks if the two lines ([param from_a], [param dir_a]) and ([param from_b], [param dir_b]) intersect. If yes, return the point of intersection as [Vector2]. If no intersection takes place, returns [code]null[/code].
-				[b]Note:[/b] The lines are specified using direction vectors, not end points.
+				Checks if the two lines ([param from_a], [param dir_a]) and ([param from_b], [param dir_b]) intersect. Returns a [Vector2], or [code]null[/code] if the lines are parallel.
```

Comment:
I think I can 1-up you on this one.

The description in the current PR no longer states what the return value _is_ anymore.
```suggestion
				Returns the point of intersection between the two lines ([param from_a], [param dir_a]) and ([param from_b], [param dir_b]). Returns a [Vector2], or [code]null[/code] if the lines are parallel.
```

Reply:
Yeah that's better. The current 4.2 docs have it as "Checks if" which wasn't the confusing bit so I focused on the later stuff. I do like having the up front though as it's more in line w/ style